company/models.py

```diff
@@ -25,8 +25,8 @@ def company_count(self):
     def __str__(self):
         return self.name
 
-    def save(self, force_insert=False, force_update=False):
-        super(Industry, self).save(force_insert, force_update)
+    def save(self, *args, **kwargs):
+        super(Industry, self).save(*args, **kwargs)
```

Comment:
Plz remove redundant Industry and self

Reply:
and do it for all other changed

Reply:
excuse me for that

Reply:
one last requirement :) please make all of your changes in one commit

Reply:
sorry if i did it wrong at first
and thanks again for helping me to do this in a right way
is it ok now ?

Reply:
thanks for your effort :)